RecoParticleFlow/PFClusterTools/src/PFEnergyCalibration.cc

```diff
@@ -801,8 +804,29 @@ PFEnergyCalibration::EcorrPS(double eEcal,double ePS1,double ePS2,double etaEcal
 
   // gives the good weights to each subdetector
   double gammaprime=Gamma(etaEcal)/9e-5;
-  outputPS1=gammaprime*ePS1;
-  outputPS2=gammaprime*Alpha(etaEcal)*ePS2;
+
+  if(outputPS1 == 0 && outputPS2 == 0 && esEEInterCalib_ != 0){
+    //    std::cout << " >>> both working " << std::endl;
```

Comment:
Please remove commented out code.
If you think they are needed, change to LogTrace or LogDebug